packages/engine-ui/src/components/Notifications.tsx

```diff
@@ -75,14 +75,15 @@ const Notifications: React.FC<{
   const withLinkElement = (
     id: string,
     entityType: EntityType,
-    text: string
+    text: string,
+    name?: string
   ) => {
     const path = entityType === EntityType.BOOKING ? 'bookings' : 'transports'
     return (
       <>
         {text}
         <Elements.Links.RoundedLink margin="0 0.5rem" to={`/${path}/${id}`}>
-          {helpers.formatIdForEndUser(id)}
+          {name || helpers.formatIdForEndUser(id)}
```

Comment:
Maybe just remove the formatting outside of the function instead of adding a conditional?